### PR TITLE
BACKLOG-19843 Fix cursor, update selection, adjust structured query f…

### PR DESCRIPTION
**Pull request by AKarmanov**
…or contents

## JIRA

https://jira.jahia.org/browse/BACKLOG-19843

## Description
Add cursor for selectable and double clickable rows 
Fix selection reducer 
Update structured query parameters for contents query
Added selection caption, commented out effect which I don't think we need but not 100% sure just yet

### CODE REVIEW

File: src/javascript/SelectorTypes/Picker/PickerDialog/PickerDialog2.jsx [hunk 1]
```diff
@@ -162,6 +164,17 @@ export const PickerDialog = ({
         }
     }, [dispatch, editorContext, isOpen, initialSelectedItem, pickerConfig, selectedItem, state, currentFolderInfo]);
 
+    useEffect(() => {
+        if (isOpen && inputContext.actionContext.fieldData) {
```
Comment (tdraier) [inline src/javascript/SelectorTypes/Picker/PickerDialog/PickerDialog2.jsx:168]: can't we rely on the "initialSelectedItem" ?

File: src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx [hunk 2]
```diff
@@ -113,9 +114,10 @@ export const ContentTable = ({
 
     useEffect(() => {
         if (selection.length > 0) {
-            const toRemove = selection.filter(s => paths.indexOf(s.path) === -1).map(s => s.path);
+            const toRemove = selection.filter(s => paths.indexOf(s.path) === -1);
             if (toRemove.length > 0) {
-                dispatch(reduxActions.removeSelectionAction(toRemove));
+                // TODO why do we have this?
+                // dispatch(reduxActions.removeSelectionAction(toRemove));
```
Comment (tdraier) [inline src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx:120]: That was something in jcontent, to remove selection (preview) if an item is not present in the list anymore. But that does not make sense in picker (where selection is persistent)

**Review by tdraier — APPROVED** [review]

File: src/javascript/SelectorTypes/Picker/PickerDialog/PickerDialog2.jsx (hunk 1 above)
Comment (AKarmanov) [reply src/javascript/SelectorTypes/Picker/PickerDialog/PickerDialog2.jsx:168]: Fileddata comes from usePickerInputData hook which we have in configs, I think it should be pretty reliable. There can be an error when it is not found in which case fieldData is not set.

File: src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx (hunk 2 above)
Comment (AKarmanov) [reply src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx:120]: Ok, I will remove it then.

**Pull request merged**